facebook/draft-js
# Facebook#1077 - Fix for input bug on Android - space or a special char clears the input

**Summary**

Fix for input bug on Android - typing space or a special char clears the input
https://github.com/facebook/draft-js/issues/1077

**Test Plan**
1. Add the component to your React website (test website)
2. Run Android emulator
3. Open your test website
4. Type a word into the component (for example, "All") and push space
5. Type another word (for example, "right") and push space

Expected behavior:
Text is not cleared, autocomplete is applied as expected

Wrong behavior fixed by this change:
Text is cleared or wrong symbols are added
